decidim-core/app/cells/decidim/content_blocks/participatory_space_stats_cell.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ContentBlocks
+    class ParticipatorySpaceStatsCell < BaseCell
+      def show
+        return if stats.blank?
+
+        render
+      end
+
+      private
+
+      def stats; end
```

Comment:
Is this pending?

Reply:
This method is overridden by classes that inherit from it, in this case in `decidim-participatory_processes/app/cells/decidim/participatory_processes/content_blocks/stats_cell.rb` If the method is not overriden there is not an error, it appears a no stats message, but I can add in the method something like that to force an error: 
```ruby
raise "#{self.class.name} is expected to implement #stats"
``